Documentation/Editor/Index.rst

```diff
@@ -0,0 +1,50 @@
+.. include:: ../Includes.txt
+
+
+.. _for-editors:
+
+===========
+For Editors
+===========
+
+Target group: **Editors**
+
+What is a widget?
+=================
+
+A widget consists of a canvas with a given size, which is enriched with data over time.
```

Comment:
I guess this is a bit too technical. Maybe just say something like:

A widget is an element on the dashboard that displays up to date information or gives you the possibility to easily execute a command.